groundstation/static/js/components/AutomatedCommandSequence.jsx

```diff
@@ -0,0 +1,299 @@
+        if ((idx == 0 && up == true) || (idx == this.state.automatedcommands.length - 1 && up == false)) {
+            return;
+        } 
+        
+        let offset = (up == true) ? -1 : 1;
+        let first_url = '/api/automatedcommands/' + this.state.automatedcommands[idx].automatedcommand_id;
+        let second_url = '/api/automatedcommands/' + this.state.automatedcommands[idx + offset].automatedcommand_id;
+
+        let first_req = {
+            method: "PATCH",
+            headers: {
+                "Content-Type": "application/json",
+                "Authorization": "Bearer " + localStorage.getItem("auth_token")
+            },
+            body: JSON.stringify({priority: idx + offset})
+        };
+        let second_req = {...first_req, body: JSON.stringify({priority: idx})};
+
+        Promise.all([
+            fetch(first_url, first_req),
+            fetch(second_url, second_req)
+        ]).then(([res1, res2]) => {
+            return Promise.all([res1.json(), res2.json()])
+        }).then(([res1, res2]) => {
+            if (res1.status == "success" && res2.status == "success") {
+                const obj = this.state.automatedcommands.slice();
+                obj[idx + offset] = res1.data;
+                obj[idx] = res2.data;
+                this.setState({automatedcommands: obj});
+            }
+        });
+    }
+
+    render() {
+        return (
+            <div>
+            <Paper className="grid-containers" style={{marginBottom: '20px'}}>
+            <Grid container>
+                <Grid item xs={11}>
+                    <Typography variant="h5" style={{padding: '10px'}}>Automated Command Sequence</Typography>
+                </Grid>
+				<Grid item xs={1} style={{textAlign: 'right'}}>
+					<Fab 
+                    disabled={!this.state.is_admin} 
+                    style={{position: 'inherit'}} 
+                    onClick={ (event) => this.handleAddCommandOpenClick(event) }>
+						<AddIcon 
+								style={{ color: '#4bacb8', fontSize: '2rem'}} 
+						/>
+					</Fab>
+				</Grid>                
+            </Grid>    
+            </Paper>
+
+            <Grid container style={{paddingBottom: '12px', justifyContent: 'center'}}>
+            <Paper className="grid-containers" style={{minWidth: 650}}>
+                <Grid item>
+                    <AutomatedCommandSequenceList
+                    isLoading={this.state.isLoading}
+                    commands={this.state.automatedcommands}
+                    handleRearrangeClick={this.handleRearrangeClick}
+                    handleDeleteCommandOpenClick={this.handleDeleteCommandOpenClick}
+                    empty={this.state.empty}
+                    is_admin={this.state.is_admin}
+                    />
+                </Grid>
+            </Paper>                
+            </Grid>
+            
+            <Dialog open={this.state.addCommandOpen} onClose={(event) => this.handleAddCommandOpenClick(event)}>
```

Comment:
I've tried making the dialog box taller, but it looks really awkward when you haven't pressed the select yet and 80% of the dialog space is just blank. If you check out the add flight schedule dialog, the same thing happens where it's hard to scroll through the commands.

However, if you scroll down to the "optional sizes" section at this [link](https://mui.com/material-ui/react-dialog/), they have a dialog with a select inside where the select goes over the dialog box. I'd have to use the MUI select component instead of the react-select component though. If you like the design of this, I could try implementing it.